salt/journal-cms/scripts/restore-legacy.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+# restore legacy files. run as root
+set -exv
+
+# download the legacy files
+cd /opt/ubr/
+./ubr.sh download s3 adhoc _platform.sh/legacy_cms.sql.gz _platform.sh/legacy_cms_files.tar.gz
+
+# the database can be handled with ubr
+./ubr.sh restore file adhoc /tmp/ubr/legacy_cms.sql.gz mysql-database.legacy_cms
```

Comment:
does this mean that `./ubr.sh restore` will run the SQL contained in `legacy_cms.sql.gz`. If so ok, but where did it get the configuration about the host, username and password to do so?